pkg/odo/cli/config/view.go

```diff
@@ -53,6 +54,9 @@ func (o *ViewOptions) Complete(name string, cmd *cobra.Command, args []string) (
 
 // Validate validates the ViewOptions based on completed values
 func (o *ViewOptions) Validate() (err error) {
+	if !o.IsDevfile {
```

Comment:
It would be better to use the `genericclioptions.New` function in `Complete`, that will do all these checks for you

Reply:
Updated!, Thanks @feloy **:)**